corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -324,6 +337,7 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
             this.search = null;
             this.queryData = null;
             this.sessionId = null;
+            Utils.clearQueryInputs();
```

Comment:
Should we make this a `clearQueryData` and throw `this.queryData` in there as well?

Reply:
I realized that clearing the query inputs actually breaks sticky query inputs. That data was being stored whether or not the feature flag was enabled -- the flag just controls whether it's pulled from or not. So I reverted that commit. https://github.com/dimagi/commcare-hq/pull/33946/commits/5669124cef39ec39045407913fb5d93bbd59bb34

Reply:
that makes sense